test/resources/addon1.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: addon1
+  namespace: openshift-pipelines-operator
```

Comment:
the namespace here should be `tekton-pipelines`
as the `tekton-pipelines-controller`, `tekton-pipelines-webhook` and `addon1` expected to appear in `tekton-pipelines` namespace